static/src/js/util/humanizedQueryValueFormatters.js

```diff
@@ -79,3 +79,43 @@ export const formatFacetHierarchy = (value, order) => value.map(
     return hierarchy
   }
 )
+
+/**
+ * Humanizes the sort key value sent to CMR with a human readable value
+ * @param {String} sortKey - The value for the sort key type sent to CMR
+ * @param {String} sortsArray - The array containing all of the sort orders for the concept type
+ * @returns {String} A humanized human readable string to display
+ */
+export const humanizeSortKey = (sortKey, sortsArray) => {
+  if (!sortKey) return 'Unknown'
+
+  // Define mapping objects for collection and granule sort keys
+  const collectionSortKeyMap = {
+    '-score': 'Relevance',
+    '-usage_score': 'Usage',
+    '-ongoing': 'End Date',
```

Comment:
I do wonder if it is a problem that we are calling out oldest v. newest on the granules list but not here. Im tempted to say we should clarify but I dont want to confuse or set the wrong expectations. Leaning more towards adding the clarification. Any opinion @abbottry? It might be something worth calling attention to during demos at least.

Reply:
My 2 cents would be that we'd allow directionality on collection sort which is a recent improvement ticket I wrote. The only thing with that is that it is a bit awkward since it'd get us least used or least relevant collections which just seems like a useless thing to have, and add unnecessary confusion